<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="11" hostname="INAS-LP024" name="com.vrt.testcases.setup_CalculationTest" tests="80" failures="5" timestamp="2021-03-25T04:25:00 IST" time="3008.622" errors="0">
  <testcase name="CAL014" time="35.544" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL034" time="41.114" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL018A" time="39.156" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL011A" time="35.746" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	 Fail: Minimum value 0.10 is not allowed in D Value field expected [0.10] but found [1.00]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
 Fail: Minimum value 0.10 is not allowed in D Value field expected [0.10] but found [1.00]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.vrt.testcases.setup_CalculationTest.CAL011A(setup_CalculationTest.java:426)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL011A -->
  <system-out/>
  <testcase name="CAL011A" time="34.963" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011A -->
  <system-out/>
  <testcase name="CAL008B" time="35.458" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL007" time="39.076" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL016A" time="35.286" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL032" time="35.355" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL013A" time="38.266" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL008" time="36.732" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL011B" time="37.592" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Fail:Minimum value 99.99 is not allowed in D Value field  expected [99.99] but found [1.00]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Fail:Minimum value 99.99 is not allowed in D Value field  expected [99.99] but found [1.00]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.vrt.testcases.setup_CalculationTest.CAL011B(setup_CalculationTest.java:444)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL011B -->
  <system-out/>
  <testcase name="CAL031" time="35.481" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL009" time="35.420" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL017" time="39.219" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL011B" time="37.510" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011B -->
  <system-out/>
  <testcase name="CAL011B" time="37.431" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011B -->
  <system-out/>
  <testcase name="CAL016B" time="35.637" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL004" time="34.831" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL013A" time="39.012" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL018B" time="39.905" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL010" time="35.066" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	 Fail:the default value is not displayed in BD Value field  expected [5] but found [4]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
 Fail:the default value is not displayed in BD Value field  expected [5] but found [4]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.vrt.testcases.setup_CalculationTest.CAL010(setup_CalculationTest.java:410)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL010 -->
  <system-out/>
  <testcase name="CAL042" time="36.091" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="SC106" time="35.478" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL037" time="35.771" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL008" time="36.120" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL018A" time="39.004" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL006" time="35.698" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL013A" time="39.283" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL018A" time="39.163" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="SC110" time="36.105" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL016A" time="35.731" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL017" time="38.907" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL011B" time="36.306" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011B -->
  <system-out/>
  <testcase name="CAL008A" time="36.214" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL019" time="34.977" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL033" time="41.972" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL012" time="39.395" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL018A" time="38.723" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL018C" time="35.479" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL003" time="34.957" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL015" time="35.695" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL001" time="34.816" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL016B" time="35.872" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL045" time="43.935" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL035" time="38.161" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL018A" time="39.197" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL010" time="35.756" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL010 -->
  <system-out/>
  <testcase name="CAL011B" time="36.695" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011B -->
  <system-out/>
  <testcase name="CAL016B" time="36.254" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL011A" time="35.657" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011A -->
  <system-out/>
  <testcase name="CAL012" time="39.141" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL013A" time="39.456" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL011B" time="37.253" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Fail:Minimum value 99.99 is not allowed in D Value field  expected [99.99] but found [1.00]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Fail:Minimum value 99.99 is not allowed in D Value field  expected [99.99] but found [1.00]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.vrt.testcases.setup_CalculationTest.CAL011B(setup_CalculationTest.java:444)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL011B -->
  <system-out/>
  <testcase name="CAL013B" time="39.488" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL011B" time="36.675" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011B -->
  <system-out/>
  <testcase name="CAL011B" time="37.611" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL011B -->
  <system-out/>
  <testcase name="CAL008" time="36.299" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL005" time="35.741" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL010" time="35.061" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL010 -->
  <system-out/>
  <testcase name="CAL013A" time="38.748" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL011B" time="36.921" classname="com.vrt.testcases.setup_CalculationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Fail:Minimum value 99.99 is not allowed in D Value field  expected [99.99] but found [1.00]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Fail:Minimum value 99.99 is not allowed in D Value field  expected [99.99] but found [1.00]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.vrt.testcases.setup_CalculationTest.CAL011B(setup_CalculationTest.java:444)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CAL011B -->
  <system-out/>
  <testcase name="CAL007" time="38.973" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL029" time="42.614" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL041" time="37.525" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL039" time="38.077" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL013A" time="38.502" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL006A" time="35.702" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL007" time="39.988" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL002" time="34.510" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL016B" time="36.193" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL012" time="39.794" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL018A" time="38.577" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL041" time="59.257" classname="com.vrt.testcases.setup_CalculationTest">
    <skipped/>
  </testcase> <!-- CAL041 -->
  <system-out/>
  <testcase name="CAL007" time="39.695" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL038" time="38.332" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL030" time="34.756" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL012" time="38.261" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL016A" time="36.145" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
  <testcase name="CAL036" time="38.115" classname="com.vrt.testcases.setup_CalculationTest"/>
  <system-out/>
</testsuite> <!-- com.vrt.testcases.setup_CalculationTest -->
